src/routes/RemediationDetails.js

```diff
@@ -131,70 +133,39 @@ class RemediationDetails extends Component {
-                                            Playbook details
-                                        </CardHeader>
-                                        <CardBody>
-                                            <Stack>
-                                                <StackItem>Created by: { formatUser(remediation.created_by) }</StackItem>
-                                                <StackItem>Created: { formatDate(remediation.created_at) }</StackItem>
-                                                <StackItem>Last modified by: { formatUser(remediation.updated_by) }</StackItem>
-                                                <StackItem>Last modified: { formatDate(remediation.updated_at) }</StackItem>
-                                                {
-                                                    isBeta &&
-                                                    <StackItem className='ins-m-border-top'> Shared with: unknown </StackItem>
-                                                }
-                                            </Stack>
-                                        </CardBody>
-                                    </Card>
-                                </GridItem>
-                            </Grid>
+                            <Card>
+                                <CardHeader className='ins-m-card__header-bold'>Playbook Summary</CardHeader>
+                                <CardBody>
+                                    <Flex className='ins-c-playbookSummary' breakpointMods={ [{ modifier: FlexModifiers.column }] }>
+                                        <Flex className='ins-c-playbookSummary__overview'>
+                                            <FlexItem breakpointMods={ [{ modifier: FlexModifiers['spacer-xl'] }] }>
+                                                <DescriptionList
+                                                    isBold
+                                                    title='Total systems'>
+                                                    { pluralize(totalSystems, 'system') }
+                                                </DescriptionList>
+                                            </FlexItem>
+                                        </Flex>
+                                        <DescriptionList className='ins-c-playbookSummary__settings' title='Playbook settings'>
+                                            <Flex>
+                                                <FlexItem breakpointMods={ [{ modifier: FlexModifiers['spacer-xl'] }] }>
+                                                    Autoreboot:&nbsp;
+                                                    <b> { remediation.auto_reboot && remediation.needs_reboot ? 'Enabled' : 'Disabled' } </b>
+                                                </FlexItem>
+                                                <FlexItem>{ pluralize(stats.systemsWithReboot, 'system') } require reboot</FlexItem>
```

Comment:
what was the conclusion on require vs requires? Do we not pluralize to that level of detail or should this be fixed?

Reply:
It's being fixed. Thanks for finding it!